services/translation.py

```diff
@@ -11,16 +8,16 @@
 
 def fetch_translated_text(settings: Settings, text: List[str]):
     result = GoogleTranslateReader(settings=settings).translate(text)
-    return unsafe_perform_io(result).unwrap()
+    return result
```

Comment:
@idisblueflash I've gotten rid of `unsafe_perform_io` and `@impure_safe` because that library is hiding errors, it's not clear where something failed, so we'll use regular exceptions now